submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" >
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" >
+		<title>Document</title>
+		<link rel="stylesheet" href="style.css" >
+	</head>
+	<body>
+		<div class="wrapper">
+			<div class="header">
+				<nav>
+					<ul class="menu-list">
+						<li>
+							<a class="aaa" href="">Mail</a>
+						</li>
+						<li>
+							<a href="">Image</a>
+						</li>
+						<li class="focuses">
+							<label for="navbar-button" class="navbar-button">
+								<img class="imagePoup" src="Image/popup-button.png" alt="" >
```

Comment:
Here, and in other places, don't use camel case in class names. Use kebab case.
```suggestion
								<img class="image-poup" src="Image/popup-button.png" alt="" >
```